.github/workflows/test.yml

```diff
@@ -132,8 +142,10 @@ jobs:
       - name: Run Tests
         run: go test github_test.go integrationutils.go commands.go -v -race -timeout 30m -cover
         env:
-          JF_URL: ${{ secrets.PLATFORM_URL }}
-          JF_ACCESS_TOKEN: ${{ secrets.PLATFORM_ADMIN_TOKEN }}
+          # If the action is triggered by 'workflow_dispatch' the credentials are: DEV_MASTER_PLATFORM_URL & DEV_MASTER_PLATFORM_ADMIN_TOKEN
+          # If the action is triggered by a 'push' event the credentials are: PLATFORM_URL & PLATFORM_ADMIN_TOKEN
```

Comment:
*push or pull request

Reply:
I was trying to follow the workflow file conventions. As I saw the second event is 'push' and not 'pull request' as seen in the following pics:
![Screenshot 2024-03-17 at 10 21 10](https://github.com/jfrog/frogbot/assets/81029514/be9d1fc9-585b-4647-909a-d72f93443d0f)

![Screenshot 2024-03-17 at 10 21 34](https://github.com/jfrog/frogbot/assets/81029514/ab9e5fc6-5b85-4684-8561-315403c7cdee)

What do you think? @sverdlov93 @yahavi

Reply:
But the first 'if' implies a pull request, @eranturgeman